rules/windows/process_creation/proc_creation_win_invoke_sharefinder_discovery.yml

```diff
@@ -0,0 +1,26 @@
+title: Invoke-ShareFinder Discovery Activity
+id: 208b13d5-3e9e-48be-b059-a89b311b88de
+status: experimental
+description: Detects the usage of ShareFinder to discover File shares using Windows-Sysmon logsource
+references:
+    - https://thedfirreport.com/2023/01/23/sharefinder-how-threat-actors-discover-file-shares/
+author: Harjot Singh, '@cyb3rjy0t'
+date: 2023/01/26
+tags:
+    - attack.discovery
+    - attack.t1135
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection1:
+        CommandLine|contains:
+            - Invoke-ShareFinder 
+    selection2:
+        CommandLine|contains:
+            # Invoke-ShareFinder Base64
+            - 'SW52b2tlLVNoYXJlRmluZGVy'
```

Comment:
The "Invoke-ShareFinder" command line is already cover https://github.com/search?q=repo%3ASigmaHQ%2Fsigma%20Invoke-ShareFinder&type=code

I will wait until #3959 is merged to add the base64 to 6385697e-9f1b-40bd-8817-f4a91f40508e and it's related rule.

Reply:
Unable to see anything under "https://github.com/search?q=repo%3ASigmaHQ%2Fsigma%20Invoke-ShareFinder&type=code"

Reply:
https://github.com/SigmaHQ/sigma/blob/1be13b3ea5fff5d5d2d846b0b7017c0453b0b1d4/rules/windows/process_creation/proc_creation_win_susp_sharpview.yml#L153

Reply:
As it is a duplicate detection ,  I remove to no block the whole PR

Reply:
The threat  explained in the blog "https://thedfirreport.com/2023/01/23/sharefinder-how-threat-actors-discover-file-shares/" can bypass the current detection "[sigma/rules/windows/process_creation/proc_creation_win_susp_sharpview.yml]" as it monitoring the commandline argument "Invoke-Sharefinder" in  Sharpview process.

The "Invoke-Sharefinder" can simply be invoked using powershell process.